docs/pjrt.md

```diff
@@ -1,26 +1,127 @@
-# Experimental PjRt Runtime Support
+# Experimental PJRT Runtime Support
 
-_This document reflects the current state of PjRt support in current nightly
+_This document reflects the current state of PJRT support in current nightly
 builds_. See the [same document on the r1.13 branch](https://github.com/pytorch/xla/blob/r1.13/docs/pjrt.md)
 for the status in the latest stable release.
 
 The PyTorch/XLA team is currently migrating from the currently-supported XRT
-runtime to the [PjRt
+runtime to the [PJRT
 runtime](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/compiler/xla/pjrt)
 used by [JAX](https://github.com/google/jax).
 
-PjRt is available as an _experimental preview_ in PyTorch/XLA r1.13. The
-PyTorch/XLA team will provide limited support on a best-effort basis during this
-preview. If you encounter a bug with PjRt, please file an issue on GitHub with
-the `runtime` tag.
+PJRT is available for preview in PyTorch/XLA r2.0. **We are planning to make
```

Comment:
we actually call PJRT stable in our blog post, maybe we should align on the naming. @miladm

Reply:
Well, it is the only feature that we could call stable otheriwise everything we introduced are preview lol.

Reply:
Discussed offline and decided to use "Beta"